docs/plugins.md

```diff
@@ -0,0 +1,267 @@
+## Plugin Support
+
+You can customize the linter with rules that are more specific to your use case with the `ember-template-lint` plugin system.
+
+Plugins can define new rules and set up default configurations that can be extended.
+
+### Adding Plugins to Your Configuration
+
+In order to enable a plugin, you must add it to the `plugins` key in your configuration file.
+
+`plugins` accepts an array of either objects, or strings that resolve to files that export plugin objects.
+
+#### Define a plugin inline
+
+```javascript
+// .template-lintrc.js
+
+module.exports = {
+  plugins: [
+    {
```

Comment:
Lets define the `Plugin` interface because its the same interface in both the inline definition and relative path definition examples.